tools/aws_cli.py

```diff
@@ -13,17 +13,20 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# pylint: disable=line-too-long
 # Make sure that your AWS credentials are configured correclty, see
-# https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html #pylint: disable=line-too-long
+# https://boto3.amazonaws.com/v1/documentation/api/latest/guide/configuration.html
 """Demo CLI tool for AWS."""
 
+# pylint: enable=line-too-long
+
 import json
 import os
 
 from datetime import datetime
 from typing import TYPE_CHECKING
 
-from libcloudforensics.providers.aws.internal import account
+from libcloudforensics.providers.aws.internal import account, iam, ec2
```

Comment:
Split this in one import statement per line (see https://github.com/google/cloud-forensics-utils/pull/302)